submissions/sprint_b/app.py

```diff
@@ -0,0 +1,29 @@
+from . import app
+from flask import jsonify
+from .helpers import EpithetGenerator, Vocabulary
+
+
+@app.route("/")
+def generate_epithets():
+    epithet_generator = EpithetGenerator()
+    final_string = epithet_generator.generate_epithets()
+
+    return jsonify({"epithets": final_string})
+
+@app.route("/<int:insults>")
```

Comment:
The rubric specifies three routes, this isn't one of them. Please update route to '/epithet/<quantity>' as requested in the rubric.